tsl/test/expected/exp_cagg_next_gen.out

```diff
@@ -152,14 +152,14 @@ NOTICE:  drop cascades to table _timescaledb_internal._hyper_6_20_chunk
 \c :TEST_DBNAME :ROLE_SUPERUSER
 CREATE DATABASE test;
```

Comment:
Hmm... this is not your code, but since you seem to make changes to avoid the problem, it might be good to consider updating this.

To avoid tests interfering with each others when executing in parallel, we have set the variable `TEST_DBNAME`. Shouldn't you use this here to avoid another test that creates a `test` database interfering with this test? This is used in, e.g., tests `bgw_launcher`, `loader-tsl`, and `pg_dump` so might be good to stick to this pattern.

Reply:
It can be material for a following PR.

Reply:
Anyway fixed for this testcase.